### ISSUE TITLE
About iphone dataset

### ISSUE THREAD

**Issue by xingy038** [facebookresearch/robust-dynrf#8]
Thanks for your valuable work. I want to know how do you train the iphone dataset? How to get the value in the paper?

**Comment by violetteshev** [comment]
Hi @xingy038, did you manage to reproduce the results on iPhone dataset? I'm looking for some guidance about configs/data preprocessing.

**Comment by xingy038** [comment]
> Hi @xingy038, did you manage to reproduce the results on iPhone dataset? I'm looking for some guidance about configs/data preprocessing.

[iphone.zip](https://github.com/facebookresearch/robust-dynrf/files/12515754/iphone.zip)
Hi, I use this code to load data. 
Actually, I just used iphone dataset to evaluate my project, so I didn't reproduce this project.

**Comment by violetteshev** [comment]
> [iphone.zip](https://github.com/facebookresearch/robust-dynrf/files/12515754/iphone.zip) Hi, I use this code to load data. Actually, I just used iphone dataset to evaluate my project, so I didn't reproduce this project.

Thanks a lot!

**Comment by quan5609** [comment]
Hi @violetteshev, do you have any updates for iPhone dataset? I am also looking for the training code for iPhone dataset. Thank you!

**Comment by violetteshev** [comment]
No, I couldn't get proper results on iPhone data with this model
